en_US/deploy/install-macOS.md

```diff
@@ -7,20 +7,20 @@ Supported versions:
 - macOS 12
 - macOS 11
 
-The section below will take macOS 12 as an example to illustrate how to download the latest version of EMQX. For other If you want to install a different version or in a different system, please visit the [EMQX Deployment page](https://www.emqx.com/zh/try?product=enterprise). 
+The section below will take macOS 12 as an example to illustrate how to download the latest version of EMQX. For other If you want to install a different version or in a different system, please visit the [EMQX Deployment page](https://www.emqx.com/en/try?product=enterprise). 
 
 {% emqxce %}
 
-1. Download [emqx-5.0.17-macos12-arm64.zip](https://www.emqx.com/en/downloads/broker/5.0.17/emqx-5.0.17-macos12-arm64.zip). 
+1. Download [emqx-5.0.19-macos12-arm64.zip](https://www.emqx.com/en/downloads/broker/5.0.19/emqx-5.0.19-macos12-arm64.zip). 
```

Comment:
add an option to use [Homebrew?](https://brew.sh/)?

```
brew install emqx
```

Reply:
Also, if you add a link to arm64 build, probably a good idea to mention that this package is for Apple Silicon macs

Reply:
@LenaLenaPan you can add it to the task list.